### DISCUSSION TITLE
Flux bootstrap gitlab error

### DISCUSSION THREAD

**Discussion by zandercreations** [fluxcd/flux2#1444] [Q&A] [answered]
Hi,

I'm getting a flux error when I try to export my gitlab token to the environment variable GITLAB_TOKEN

Any help will be appreciated.

Thanks

<img width="1586" alt="gitlab_token_error" src="https://user-images.githubusercontent.com/17857024/119246699-a9c5f500-bb51-11eb-8395-0746b0aa5fe2.png">

**Comment by hiddeco** [comment, marked as answer]
The variable you `export` is not available to `flux` if you run it using `sudo`, as this will switch contexts.

The solution is to either run `sudo -E flux [...]`, or simply run `flux` without sudo, which generally should work.
